clio-server/src/main/scala/org/broadinstitute/clio/server/dataaccess/elasticsearch/ElasticsearchQueryMapper.scala

```diff
@@ -85,6 +70,18 @@ class ElasticsearchQueryMapper[Input: ClassTag: FieldMapper] {
       case (name, _) => name
     }
   }
+
+  /**
+    * Munges the query result document into the appropriate form for the query output.
+    *
+    * @param document A query result document.
+    * @return The query output.
+    */
+  def toQueryOutput(document: Json): Json = {
+    document.mapObject(_.filterKeys({ key =>
```

Comment:
Shouldn't need the parens around the `{ }` in `filterKeys`

Reply:
Er, alt: `filterKeys(!keysToDrop.contains(_))` (might not work because of the `!`)